teeracle/src/benchmarking.rs

```diff
@@ -44,8 +44,9 @@ benchmarks! {
 		ensure_not_skipping_ra_check();
 		timestamp::Pallet::<T>::set_timestamp(TEST4_SETUP.timestamp.checked_into().unwrap());
 		let signer: T::AccountId = get_signer(TEST4_SETUP.signer_pub);
-		let currency = "usd".as_bytes().to_owned();
+		let trading_pair =  "DOT/USD".as_bytes().to_owned();
 		let rate = U32F32::from_num(43.65);
+		let data_source = "https://api.coingecko.com".as_bytes().to_owned();
```

Comment:
either you limit the size of these vec arguments in the dispatchable function or you benchmark with variable size of vecs such that longer vecs cost more.
As it is, these arguments are an attack vector

Reply:
Exactly. I guess it is more reasonable to limit the vec size.